src/sentry/receivers/outbox/region.py

```diff
@@ -75,13 +74,6 @@ def process_organization_member_updates(
     )
 
 
-@receiver(process_region_outbox, sender=OutboxCategory.TEAM_UPDATE)
-def process_team_updates(
-    object_identifier: int, payload: Any, shard_identifier: int, **kwargs: Any
-):
-    maybe_process_tombstone(Team, object_identifier)
-
```

Comment:
When these changes deploy what will happen to any queued team_update outbox messages? Should we have a no-op handler just to drain those messages out while transitioning to the replicated table?

Reply:
We are still using TEAM_UPDATE -- they aren't dropped.  The logic moved.  Notice that `Team` has `category = TEAM_UPDATE`.

Essentially, the existing logic and categories are handled exactly as before, but the logic moved into the model.